source/features/hide-comments-faster.tsx

```diff
@@ -17,6 +17,9 @@ function generateSubmenu(hideButton: Element): void {
 	const comment = hideButton.closest('.unminimized-comment')!;
 	const hideCommentForm = select('.js-comment-minimize', comment)!;
 
+	// remove the d-flex class from the comment form
```

Comment:
Comments should be a word-for-word explanation of the code, they should explain why the code exists or what it does. In this case it's not really needed since it's self-explanatory that it removes the class because it's not the behavior we want.